public/assets/js/allcount.js

```diff
@@ -54,6 +54,10 @@ allcountModule.config(["fieldRenderingServiceProvider", function (fieldRendering
             return s && moment(s, 'YYYY-MM-DD HH:mm:ss').toDate();
         }
 
+        function parseDateToMoment(s) {
```

Comment:
@stuhol Could you please clarify why do we need `parseDateToMoment` (why `parseDate` isn't enough here)?